module3/Projects/PairProjectCodeReview.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Peer Code Review
+---
+
+You will be paired with one person from _a different_ group for today's peer review session.
+
+1. Share a link to your github repository.
+2. Spend 20 minutes to review the other teams' code. Answer the following questions (write them down to share!)
+    * Where are they performing the conversion of PriceInCents to PriceInDollars?
+    * When editing an item, what kind of data is expected in the 'price' field? Are there any disadvantages to this datatype? Is there another place this conversion could be made?
+    * What differences are there in the migration files between their project and yours?
+    * How is the total spent by customer calculated?
+    * In reviewing the other group's code - what questions do you have for them, or for an instructor?
+3. Spend 20 minutes to share your answers to the questions above, and discuss the following:
```

Comment:
Maybe make it super clear here that in those 20 minutes both partners should share their notes.